### ISSUE TITLE
Attribute @normal for pbcoreAssetDate

### ISSUE THREAD

**Issue by caseyedavis12** [WGBH/PBCore2.0#58]
I suggest a new attribute for pbcoreAssetDate: @normal

This will provide a consistent format of the date, following iso 8601, which can then be represented differently in the element value. 

```
<pbcoreAssetDate datetype="broadcast" certainty="approximate" normal="19880423">April 23, 1988.</pbcoreAssetDate>
```

**Comment by awead** [comment]
Is this pattern you've seen used before?  The term "normal" seems misleading.  I would make the date repeatable and specify an encoding:

``` xml
<pbcoreAssetDate datetype="broadcast">April 23, 1988.</pbcoreAssetDate>
<pbcoreAssetDate datetype="broadcast" encoding="ISO8601">19880423</pbcoreAssetDate>
```

Update: OK, I see this is EAD based.

**Comment by AllisonAnn** [comment]
Maybe a dateType element field (nested elements), with source, ref,
version, annotation attributes, could be used to also hold an encoding
schema?

However, I think that. PBCore/Dublic Core coverage can be used for "normal"
style dates, keeping assetDate ISO8601 as a default rule.

**Comment by caseyedavis12** [comment]
Yes, the Encoded Archival Description (EAD) schema has a @normal attribute for dates. It provides a consistent format of the date as an attribute and allows for the value of the element to be presented to researchers in any format. I prefer this instead of requiring pbcore users to format the value of the pbcoreAssetDate element as "19880423." You can put data in any format in the value of the element as long as you qualify it in the attributes.

We could do

```
<pbcoreAssetDate datetype="broadcast" encoding="ISO8601" normal="19880423" certainty="approximate">April 23, 1988</pbcoreAssetDate>
```
